docs/servers/check-server-provision-logs.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Check Server Provision Logs
+intro: Sometimes you may encounter problems while provisioning your servers. Check your server' actions logs to evaluate what happened.
+links:
+    overview:
+    quickstart:
+    previous:
+        - /docs/servers/ssh-into-server
+    next:
+        - /docs/applications/add-application
+    guides:
+    related:
+    featured:
+---
+
+1. On Devopness, navigate to a project then select an environment
+1. Find the `Servers` card
+1. Click the `View` in the `Servers` card
+1. On the upper-right corner, click `ACTIONS` to see a list of actions for all `Servers`
+1. In the list of actions, find the action you want to inspect and click `LOGS` to see the list of action steps
+    > There are 3 actions related to the server provision process: `Provision`, `Provision-check` and `Add`;
+    > each action can have one of the following status: `Pending`, `In progress`, `Completed` or `Failed`.
+1. In the list of action steps, find the action you want to inspect and click `LOGS`
+1. In the card opened below the action step will be listed the logs related to that action step execution
```

Comment:
I believe the word `card` is being used here with a different meaning of the same word `card` used in steps 2 and 3, which could be misleading to the reader.
I'd suggest to be consistent and do not use the same word when referring to different UI elements.

```suggestion
1. A text area will be displayed containing the output log of the action step execution
```